src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -2615,52 +2533,52 @@
-  "botania.page.terraPick2": "To advance from rank $(4)C$(0) to rank $(4)B$(0), a full $(1)Mana Pool$(0) is required, and the cost increases by a tenfold per rank from there on in, all the way to $(4)SS$(0).$(p)The rank of the tool does not change its speed, but instead, it increases the power of its $(4)Active Ability$(0).$(p)Toggling the tool's ability can be done via right click.",
-  "botania.page.terraPick3": "While the tool is active, it enables it to break blocks in an extended area. At rank $(4)C$(0) it's a small column, which increases to a square at rank $(4)B$(0), that gets larger through the ranks. It's to note that the rank $(4)D$(0) tool does not have the active ability.$(p)Having the tool active consumes $(4)Mana$(0) from its storage passively, even if not being used to mine.",
-  "botania.page.terraPick4": "Lastly, the tool takes damage for every single block broken, so a good supply of $(4)Mana$(0) in the wielder's inventory is a must for the higher ranks, as the tool does function like normal $(1)Manasteel Tools$(0), in that it'll use $(4)Mana$(0) instead of actually taking damage, given it's made of $(1)Terrasteel$(0).$(p)It also won't despawn like regular dropped items.",
+  "botania.page.terraPick0": "The $(1)Terra Shatterer$(0) is a tool that, due to the amount of $(1)Terrasteel$(0) required for its crafting, can store absolutely ludicrous amounts of $(4)Mana$(0).$(p)The tool functions very much like a $(1)Mana Tablet$(0) in that it can absorb $(4)Mana$(0) by being tossed on a $(1)Mana Pool$(0). However, any $(4)Mana$(0) absorbed can't be released again, so it's nonviable as a storage device.",
+  "botania.page.terraPick1": "The amount of $(4)Mana$(0) stored on the tool defines its $(4)Rank$(0). Ranks progress as follows: $(4)D$(0), $(4)C$(0), $(4)B$(0), $(4)A$(0), $(4)S$(0), $(4)SS$(0); with $(4)D$(0) being the lowest rank and $(4)SS$(0) the highest.$(p)The amount of $(4)Mana$(0) required to advance from rank to rank increases exponentially as the rank does. To advance from $(4)D$(0) to $(4)C$(0), for example, a Shatterer requires just a few seconds of exposure to a $(1)Mana Pool$(0).",
+  "botania.page.terraPick2": "To advance from rank $(4)C$(0) to rank $(4)B$(0), a full $(1)Mana Pool$(0) is required, and the cost increases by a tenfold each rank from there through $(4)SS$(0).$(p)The rank of the tool does not change its speed; rather, it increases the power of its $(4)Active Ability$(0).$(p)A right-click will toggle the Shatterer's ability on or off.",
+  "botania.page.terraPick3": "When the Shatterer's ability is active, it can break blocks in a wider surface area. At rank $(4)C$(0) it breaks in a narrow column, which increases to a square at rank $(4)B$(0); surface areas increase as ranks do. Note that the rank-$(4)D$(0) tool does not have an ability.$(p)Having the tool active consumes $(4)Mana$(0) from its internal storage passively, even if not being used to mine.",
```

Comment:
Actually, if the tool is active and you are actively mining, it should not use any mana from its internal storage.